Mage/src/main/java/mage/target/targetadjustment/BlankTargetAdjuster.java

```diff
@@ -0,0 +1,18 @@
+package mage.target.targetadjustment;
+
+import mage.abilities.Ability;
+import mage.game.Game;
+
+/**
+ * Use this to indicate that the targets are dynamically added via the Ability directly
+ * While still satisfying the Verify target check
+ *
+ * @author notgreat
+ */
+public enum BlankTargetAdjuster implements TargetAdjuster{
```

Comment:
Maybe it's better to rename like `AutoTargetedTargetAdjuster` or `DefineByTriggerTargetAdjuster`